tests/test_api.py

```
@@ -55,9 +55,19 @@ def test_list_models():
     assert "sam" in data["models"]
     assert "sam2" in data["models"]
     assert "sam3" in data["models"]
+    assert "facebook/sam3.1" in data["models"]["sam3"]
     assert data["loaded"] == []
 
 
+def test_private_model_constant_aliases_remain_available():
+    """Legacy private constants still point to the shared registry values."""
+    import samgeo.api as api
```

Ruff (0.15.17)

[warning] Use `from samgeo import api` in lieu of alias

Replace with `from samgeo import api`

(PLR0402)